src/sentry/api/base.py

```diff
@@ -160,6 +161,20 @@ def load_json_body(self, request):
         except json.JSONDecodeError:
             return
 
+    def _create_api_access_log(self):
+        """
+        Create a log entry to be used for api metrics gathering
+        """
+        token_type = getattr(self.request.auth, "__class__", None)
+        log_metrics = dict(
+            method=self.request.method,
+            view=str(self.request.parser_context["view"].__class__),
```

Comment:
Is there ever a case where `"view"` isn't present?

Reply:
Not sure. If I add a try/catch we can find out :)